### ISSUE TITLE
Updated members list

### ISSUE THREAD

**Issue by rossabaker** [scalacenter/advisoryboard#114]
I hadn't seen the announcement that a community member spot on the Advisory Board had been filled.  The most recent minutes only mention invitations had been sent, and the attendees.md file in this repo is stale.  It would be helpful to have a reference of who is currently on the board, particularly for community members who wish to raise concerns.

**Comment by SethTisue** [comment]
PR #115 addresses this. a new set of advisory board minutes will be along within the next couple weeks or so

**Comment by SethTisue** [comment]
https://github.com/scala/scala.epfl.ch/issues/56 is out of scope for me to address personally, but I would certainly like to see it addressed.

**Comment by ckipp01** [comment]
Thanks for brining this up @rossabaker. Just to make sure everything is tied together. I went ahead and created https://github.com/scala/scala.epfl.ch/issues/259. I'm actually not fully in favor of https://github.com/scala/scala.epfl.ch/issues/56, so I figured I'd create a ticket outlining a subset of that, which is more relevant to the community imo. After #115 and #116 the attendees document is up-to-date. I'll go ahead and close this since I believe the main points are addressed, but we can further improve on this. Let's shift to the newly created issue since the changes would need to happen there.
